.gitignore

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
I think this is something better to put in `$HOME/.gitignore` rather than every repo.